usefulUtilities/sitV2/appResources/appData/sitClientV2.js

```diff
@@ -126,6 +127,20 @@
     }
 
 
+    // Save the current avatar position
+    function saveCurrentAvatarPosition(){
+        Settings.setValue(SETTING_KEY_AVATAR_POSITION, MyAvatar.position);
+    }
+
+
+    // Move the avatar back to the previous Y
+    function returnAvatarToTheCorrectY(){
+        var oldPosition = Settings.getValue(SETTING_KEY_AVATAR_POSITION, MyAvatar.position);
+        var newPosition = [MyAvatar.position.x, oldPosition.y, MyAvatar.position.z];
```

Comment:
Instead of having to read from `MyAvatar.position` twice, please set a local `var` to `MyAvatar.position` and then reference that var here.